libs/s25main/figures/nofWorkman.cpp

```diff
@@ -100,13 +100,16 @@ struct NodeHasResource
 
 MapPoint nofWorkman::FindPointWithResource(ResourceType type) const
 {
-    // Alle Punkte durchgehen, bis man einen findet, wo man graben kann
+    return FindPointWithResource(type, MINER_RADIUS);
+}
+
+MapPoint nofWorkman::FindPointWithResource(ResourceType type, unsigned radius) const {
     const std::vector<MapPoint> pts =
-      world->GetMatchingPointsInRadius<1>(pos, MINER_RADIUS, NodeHasResource(*world, type), true);
+      world->GetMatchingPointsInRadius<1>(pos, radius, NodeHasResource(*world, type), true);
     if(!pts.empty())
         return pts.front();
 
     workplace->OnOutOfResources();
```

Comment:
As you noticed earlier: This likely shouldn't be done here, but at the call site. You can (don't need to) change this, when touching the code already, especially if it bites you in the a** while working on your stuff ;-)

I often do such drive-by-improvements myself, but it is hard to not do too many changes once you start...

Just as a note, no need to change this (now).